tests/Agent/IntegrationTests/UnboundedIntegrationTests/MySql/MySqlAsyncTests.cs

```diff
@@ -125,4 +125,22 @@ public void Test()
             );
         }
     }
+
```

Comment:
I suggest expanding test coverage to the lowest/highest supported versions (remember the versions specified in the MultiFunctionApplicationHelpers.csproj)